src/main/resources/assets/memoryusagescreen/lang/ru_ru.json

```
@@ -1,8 +1,8 @@
 {
-  "text.autoconfig.memoryusagescreen.title": "Настройки Memory Usage Screen",
-  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на экране загрузки игры",
-  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира",
-  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Режим переключения отображения по нажатию",
+  "text.autoconfig.memoryusagescreen.title": "Конфигурация Memory Usage Screen",
+  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на начальном загрузочном экране.",
+  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира.",
```

Исходный текст, как и описания большинства других модов на Fabric, не содержат точек во всплывающих подсказках. 
Кроме того, точки принято оставлять исключительно в статичных подсказках или описаниях, всплывающие подсказки в соответствии с **единым стилем перевода интерфейсов программ** не должны содержать точек, если они состоят из одного предложения.
```suggestion
  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира",
```

---

В параметрах не ставятся точки это да, тут я не заметил, а вот в описании параметра должна стоять точка, по правилам написания текста.